inst/@sym/charpoly.m

```diff
@@ -54,11 +54,17 @@
   if (nargin >= 3)
     print_usage ();
   end
+
+  cmd = {'return _ins[0].charpoly(_ins[1]).as_expr(),'};
+
   if (nargin == 1)
-    error('Charpoly as vector its not supported now');
+    syms b
+    y = python_cmd(cmd , sym(a), b);
```

Comment:
does this cause some problem if `b` is in the matrix already?  Also, given #543, maybe we should just do some python code here.  Then we can use `b = Dummy()` for the temp variable.

Or better yet, there must be a command other than `as_expr` which extracts the coefficients into a list...